static_health/stackstate_checks/static_health/static_health.py

```diff
@@ -0,0 +1,76 @@
+# (C) StackState 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import csv
+import codecs
+from schematics import Model
+from stackstate_checks.base.utils.schemas import StrictStringType
+from stackstate_checks.base import ConfigurationError, AgentCheck, TopologyInstance,\
+    HealthStream, HealthStreamUrn, HealthType
+
+
+class InstanceInfo(Model):
+    type = StrictStringType(required=True, choices=["csv"])
+    health_file = StrictStringType(required=True)
+    delimiter = StrictStringType(required=True)
+
+
+class StaticHealthCheck(AgentCheck):
+    SERVICE_CHECK_NAME = "StaticHealth"
+    INSTANCE_TYPE = "StaticHealth"
+    INSTANCE_SCHEMA = InstanceInfo
+
+    def get_health_stream(self, instance_info):
+        return HealthStream(HealthStreamUrn(self.INSTANCE_TYPE, str(instance_info.health_file)))
```

Comment:
Seeing these two next two each other, I wonder whether the API should be more consistent. Or if we should call it `HealthStreamInstance` maybe, open question?

Reply:
lets dicuss